guides/source/autoloading_and_reloading_constants.md

```diff
@@ -488,6 +488,48 @@ INFO. `autoload_paths` is computed and cached during the initialization process.
 The application needs to be restarted to reflect any changes in the directory
 structure.
 
+eager_load_paths
+----------------
+
+Eager loading is the process of loading all application classes, executed once on application boot
+when cache classes is enabled (e.g. in production).
+
+Imagine an app with `Tools` module defined in `lib/tools.rb`, and `lib/` added to `autoload_paths`.
+It works as expected in development:
+
+```
+$ rails runner -e development "puts Tools"
+Tools
+```
+
+But in production, code listed in `autoload_paths` will not be loaded:
+
+```
+$ rails runner -e production "puts Tools"
+uninitialized constant Tools
+```
+
+Use `eager_load_paths` to declare directories to be autoloaded in development and eager loaded in production.
```

Comment:
this is confusing, saying it does different things in development vs production, can we rephrase it?